packages/block-library/src/list/split-on-line-break.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { last } from 'lodash';
+
+/**
+ * Split the content of a paragraph on line breaks ('<br>') into sets of
+ * content, each representing a list item.
+ *
+ * The term "content" refers to a rich-text description in the form of a mixed
+ * array of strings and element-like objects (e.g. `{ type: strong, props: {
+ * children: […] } }`).
+ *
+ * @param {Array}    fragments Rich-text content
```

Comment:
Minor: Should we be more specific and document that this function receives a WPBlockChildren parameter?

Reply:
Ah, that's a change I missed — good to know. :)

Reply:
Added.